### PR TITLE
Add test for config-manager --save with --config option

### PR DESCRIPTION
**Pull request by pkratoch**
Tests https://github.com/rpm-software-management/dnf-plugins-core/pull/373

### CODE REVIEW

**Comment by kontura** [comment]
@pkratoch can you rebase?
And also add the @not.with_os=rhel__eq__8? In case we need to run the gating again. :slightly_smiling_face:

**Comment by pkratoch** [comment]
Thanks, fixed.

File: dnf-behave-tests/features/plugins-core/config-manager.feature [hunk 1]
```diff
@@ -167,3 +167,19 @@ Scenario: --setopt does not modify repo when used without --save and one argumen
...
+    And I create file "alternative.conf" with
+        """
+        [main]
+        """
+   When I execute dnf with args "config-manager --config={context.dnf.installroot}/alternative.conf --save --setopt=debuglevel=7"
+   Then the exit code is 0
+    And file "alternative.conf" contains lines
+        """
+        [main]
+        debuglevel=7
```
Comment (kontura) [inline dnf-behave-tests/features/plugins-core/config-manager.feature]: One more thing, there are missing `"""` at the end.

Comment (pkratoch) [reply dnf-behave-tests/features/plugins-core/config-manager.feature]: Oops, thanks for the catch. Fixed.

**Pull request merged**